### DISCUSSION TITLE
Turn signals

### DISCUSSION THREAD

**Discussion by Hanbao-Deguo** [Stephan-S/FS19_AutoDrive#2057] [Ideas]
It would be nice to be able to add information about turn signals in the waypoints. E.g. when I record a path and use the turn signal, AD would store this information in the first (on) and last (off) waypoint when I use the turn signal. I know there is a setting in AD where I can turn it on when I drive a curve of an angle of at least ... degrees, but there are 90 degree curves without crossings, too.

**Comment by Axel32019** [comment]
@Hanbao-Deguo Hier schaut nicht wirklich jemand rein, daher bitte Feature-Request Issue erstellen.
